docs/about/architecture.md

```diff
@@ -72,3 +71,92 @@ while those that don't are called _peripheral resources_.
+
+![Architecture Diagram](../images/architecture-single-region.png)
+
+#### Components
+
+This architecture consists of a single load balancer, several _Coder Server_
+replicas, and _Coder workspaces_ deployed in the same region.
+
+##### Workload resources
+
+- Use Terraform to deploy at least one **Coder Server Replica** per availability
+  zone with _Coder Server_ instances and provisioners. High availability is
+  recommended but not essential for small deployments.
+- Single replica deployment is a special case that can address a
+  tiny/small/proof-of-concept installation on a single virtual machine serving
+  less than 100 workspace users.
+
+**Coder workspace**
+
+- For small deployments consider a lightweight workspace runtime like
+  [Sysbox](https://github.com/nestybox/sysbox) container runtime. Learn more how
+  to enable
+  [docker-in-docker using Sysbox](https://asciinema.org/a/kkTmOxl8DhEZiM2fLZNFlYzbo?speed=2).
+
+**HA Database**
+
+- Enable replication across all availability zones.
+- Monitor replication lag, node status, and resource utilization metrics.
+- Implement robust backup and disaster recovery strategies to protect against
+  data loss.
+
+##### Workload supporting resources
+
+**Load balancer**
+
+- Distributes and load balances traffic from agents and clients to _Coder
+  Server_ replicas across zones.
+- Layer 7 load balancing. The load balancer can decrypt SSL traffic, and
+  re-encrypt using internal certificate.
+- Sessions persistence (sticky sessions) can be disabled as _Coder Server_
+  instances are stateless.
+- WebSocket and long-time connections must be supported.
+
+**Single sign-on**
+
+- Integrate with existing Single Sign-On (SSO) solution used within the
+  organization, OAuth 2.0 or OpenID Connect standards are supported.
+- Learn more about [Authentication in Coder](../admin/auth.md).
+
+### Multi-region architecture
+
+![Architecture Diagram](../images/architecture-multi-region.png)
+
+#### Components
+
+This architecture is for globally distributed developer teams using Coder
+workspaces on daily basis. It features a single load balancer with regionally
+deployed _Workspace Proxy Replicas_, several _Coder Server_ replicas, and _Coder
+workspaces_ provisioned in different regions.
+
+Note: The _multi-region architecture_ assumes the same deployment principles as
+the _single region architecture_, but it extends them to multi region deployment
+with workspace proxies. Provisioners deployed in regions closed to developers
+offer the fastest developer experience.
+
+##### Workload resources
+
+**Workspace proxy**
+
+- Workspace proxy offers developers the option to establish a fast relay
+  connection when accessing their workspace via SSH, a workspace application, or
+  port forwarding.
+- Dashboard connections, API calls (e.g. _list workspaces_) are not served over
+  proxies.
+- Proxies do not establish connections to the database.
+- Do not share authentication tokens between proxy instances.
```

Comment:
Change to "Proxy instances do not share authentication tokens between one another".